kustomize/components/google-cloud-operations/kustomization.yaml

```diff
@@ -17,23 +17,8 @@ kind: Component
 resources:
   - otel-collector.yaml
 patchesStrategicMerge:
-# adservice - stats, tracing
-- |-
-  apiVersion: apps/v1
-  kind: Deployment
-  metadata:
-    name: adservice
-  spec:
-    template:
-      spec:
-        containers:
-          - name: server
-            env:
-            - name: ENABLE_STATS
-              value: "1"
-            - name: ENABLE_TRACING
-              value: "1"
-# checkoutservice - stats, tracing, profiler
+# adservice - not yet implemented
```

Comment:
Because I implemented the Cloud Ops options in the Helm chart, I took the initiative to update the associated Kustomize overlay to be accurate with what we have in the repo and not misleading.